src/appleseed-max-impl/appleseeddisneymtl/appleseeddisneymtl.cpp

```diff
@@ -168,6 +169,33 @@ namespace
         ParamIdBumpTexmap
     };
 
+    void update_map_buttons(IParamBlock2* pblock)
+    {
+        if (pblock == nullptr)
+            return;
+
+        IParamMap2* map = pblock->GetMap(ParamMapIdDisney);
+        if (map == nullptr)
+            return;
+
+        const int count = pblock->NumParams();
+        for (int i=0; i < count; ++i)
+        {
+            const int param_id = pblock->IndextoID(i);
+            if (param_id == ParamIdBaseColorTexmap)
```

Comment:
Again, not sure why this exception?

Reply:
It's because base color texture button is wide and contains full name. But I'll get rid of this check I think.